frontend/src/components/ViewLinks.vue

```diff
@@ -1,77 +1,51 @@
 <template>
   <b-navbar-nav>
+    <!-- <li class="play-to-earn top-nav-links">
+      <P2EButton mainText="Play-to-earn" route="play-to-earn" />
+    </li> -->
+
+    <li class="character top-nav-links" >
```

Comment:
That's why the styles were "needed" there app-scoped! I think you can just use a div, not a list element

Reply:
![image](https://user-images.githubusercontent.com/9636557/157713671-a1d44a91-9f30-4649-9cce-6ec0e6043de7.png)
they we're part of navbar ul list , tried renaming them to div but make sense still to be list because they are part of navbar list ul li